src/prefect/blocks/webhook.py

```diff
@@ -44,9 +45,13 @@ class Webhook(Block):
         default=True,
         description="Whether to allow notifications to private URLs. Defaults to True.",
     )
+    verify: bool = Field(
+        default=True,
+        description="Whether or not to enforce a secure connection to the webhook.",
+    )
 
     def block_initialization(self):
-        self._client = AsyncClient(transport=_http_transport)
+        self._client = AsyncClient(transport=AsyncHTTPTransport(verify=self.verify))
```

Comment:
this line should conditionally switch between the already instantiated `_http_transport` and `_insecure_http_transport` classes defined above instead of instantiating a new transport object

Reply:
ah yep that makes sense

Reply:
ah my fault @westford14 - you already had this right 😄